database/postgres/dal/news.py

```diff
@@ -27,3 +27,19 @@ def fetch_news(self, order_by=None, limit=None, filters=None):
     def get_news_by_id(self, blob_id: str):
         return self.session.query(News).filter(News.blob_id == blob_id).first()
 
+
+    def get_news_by_real_id(self, news_id: str):
```

Comment:
real id? I guess it refers to our Database id. Consider renaming to "native_id" or "db_id". Naming is very crucial. Sometimes it takes me more time to come up with an alrorithm than to name all the variables and methods